.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug Report
+about: Report bugs in Azure Go SDK packages.
+---
+
+Thank you for opening an issue regarding the Azure SDK for Go.
+
+Describe your issue: What happened and what did you expect to happen?
+
+
+
+How can we reproduce your issue?
+
+
+
+Describe your environment, including:
+
+* Azure Service (e.g. compute): 
```

Comment:
Can we change this to package?  Sometimes it's not always clear which package is being used especially when there are multiple API versions.

Reply:
What do you think of replacing current line with `SDK Package (e.g. "/services/compute/mgmt/2018-04-01/compute"):`?